bot/cogs/doc/cog.py

```diff
@@ -106,8 +172,11 @@ def __init__(self, bot: Bot):
         self.bot = bot
         self.doc_symbols: Dict[str, DocItem] = {}
         self.renamed_symbols = set()
-
+        self.urls: Dict[str, TODO_PLACEHOLDER] = defaultdict(TODO_PLACEHOLDER)
+        self.objects = {}
```

Comment:
self.objects is the result dicts for the parsing class, I'm not sure here if it would be worth merging the doc_symbols dict and this and then using type checks when fetching (if it's an embed, send it, otherwise parse)